tests/test_cloudevent_functions.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+import pathlib
+
+import pytest
+
+from cloudevents.http import CloudEvent, from_http, to_binary_http, to_structured_http
+
+from functions_framework import LazyWSGIApp, create_app, exceptions
+
+TEST_FUNCTIONS_DIR = pathlib.Path(__file__).resolve().parent / "test_functions"
+
+# Python 3.5: ModuleNotFoundError does not exist
+try:
+    _ModuleNotFoundError = ModuleNotFoundError
+except:
+    _ModuleNotFoundError = ImportError
+
+
+@pytest.fixture
+def cloudevent_1_0():
+    attributes = {
+        "id": "my-id",
+        "source": "from-galaxy-far-far-away",
+        "time": "tomorrow",
```

Comment:
invalid CE `time` value. Must be a timestamp.

https://github.com/cloudevents/spec/blob/master/spec.md#time

Does this not cause an error when creating the CloudEvent?

Reply:
we weren't checking the format of the time attribute. There could be many ways of formatting time (utc, ISO, etc) so this would require another minor version bump in sdk-python

Reply:
I'll delete the time attribute for now and allow the CloudEvent sdk to generate a new iso formatted timestamp using current time.

Reply:
restored time with iso string instead just to test at least one optional attribute.